src/main/webapp/utilities.js

```diff
@@ -90,3 +90,15 @@ function getDateInLocalTimeZone(dateObject = new Date()) {
   const newTime = dateObject.getTime() - dateObject.getTimezoneOffset()*60*1000;
   return new Date(newTime);
 }
+
+/**
+ * Given a list of phrases, encode them for a query string with commas
+ * separating the values and quotes surrounding the phrases
+ *
+ * @param {string[]} listOfValues list of words that should be encoded.
+ * @return {string} the value for a query parameter, with each value separated
+ *     by a comma.
+ */
+function encodeListForUrl(listOfValues) {
+  return listOfValues.map((value) => encodeURI(`"${value}"`)).join();
```

Comment:
What's the value in putting quotes around the phrases? There is no value on the client, and they are ignored by the servlet.

Reply:
Good point - no need for this. Thanks!